internal/http/services/datagateway/datagateway.go

```diff
@@ -238,3 +248,69 @@ func (s *svc) doPut(w http.ResponseWriter, r *http.Request) {
 		log.Err(err).Msg("error writing body after header were set")
 	}
 }
+
+// TODO: put and post code is pretty much the same. Should be solved in a nicer way in the long run.
```

Comment:
@labkode I was pairing on this with @kulmann and while the patch works we should use the simplehttpproxy from the stdlib here ... @labkode we can look into that after this pr.